include/ChaiVM/interpreter/code-manager/code-manager.hpp

```diff
@@ -38,24 +39,45 @@ class CodeManager final {
      */
     void loadPool(std::istream &istream);
 
+    void loadKlass(std::istream &istream);
+
     void loadFunction(std::istream &istream);
 
     chsize_t getCnst(Immidiate id);
 
-    const std::string &getCnstString(Immidiate id);
+    /**
+     * Get string by immediate, i.e. its number in constant pool.
+     * @param imm Number in constant pool.
+     * @return String.
+     */
+    const std::string &getCnstStringByImm(Immidiate imm);
+
+    /**
+     * Get string by constant that provided constant pool.
+     * Usually we take constant from constant pool directly. In case of string
+     * there is just 64-bit number that encodes String. Use this method to get
+     * string by this number.
+     * @param reg_val Number from constant pool.
+     * @return String.
+     */
+    const std::string &getCnstStringByReg(chsize_t reg_val);
```

Comment:
@levBagryansky It is not obvious why this function uses "Reg". It only became clearer when you start browsing code and see how this function is used in `executor`. Consider to make name a little more obvious. Or add explanation to documentation, please (not a good option)

Reply:
@c71n93 I don't know how. Maybe you suggest smth here.

Reply:
@levBagryansky maybe `getStringByStringPoolPos(chsize_t pos)`? It's too verbose, but makes much more sense for me.